if defined?(BetterErrors)
  BetterErrors.use_pry!
end
